packages/block-library/src/post-author/index.php

```diff
@@ -15,10 +15,16 @@
  */
 function render_block_core_post_author( $attributes, $content, $block ) {
 	if ( ! isset( $block->context['postId'] ) ) {
-		return '';
+		global $wp_query;
+		if ( $wp_query && isset( $wp_query->query_vars ) && is_array( $wp_query->query_vars ) ) {
+			$query_args = wp_parse_args( $wp_query->query_vars, $query_args );
```

Comment:
Thanks for your code. I just caught this.

Undefined variable '$query_args'

Reply:
Facepalm :D Copy pasta bytes.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/40648/commits/58965eacbc79ad2cbc2742b630cafa62768e8402

Reply:
tested and it works.